Fix linear svc handling sample weights under class_weight="balanced" (#30057)
scikit-learn/scikit-learn: doc/whats_new/upcoming_changes/sklearn.svm/30057.fix.rst

> - :class:`svm.LinearSVM` now properly passes sample weights to
>   compute_class_weight under class_weight strategy 'balanced'.

Reviewer: ```suggestion
- :class:`svm.LinearSVC` now properly passes sample weights to
  :func:`utils.class_weight.compute_class_weight` when fit with
  `class_weight="balanced"`.
```
